cmd/run/run_test.go

```diff
@@ -283,6 +284,23 @@ func TestVerifyConfig(t *testing.T) {
 		err := VerifyConfig(cfg)
 		require.Error(t, err)
 	})
+
+	t.Run("correct datastore engine specified", func(t *testing.T) {
+		cfg := DefaultConfig()
+		cfg.Datastore.URI = "postgresql://localhost:5432"
```

Comment:
I didn't think `postgresql` was a valid prefix? The enumerations in `cmd/shared.go` below define `postgres`.

This test should return an error, and we should change this test to assert `postgres://localhost:5432` does not.

Reply:
I think, I accidently set URI in this test. I just wanted to test for correct DatastoreEngine. After changes, I am getting expected result. 
Although one thing which I noticed is, on setting wrong postgres URI, `RunServer` stalls and waits for connection. I think we should fail early. Let me know your thoughts